spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestMetadataTablesWithPartitionEvolution.java

```diff
@@ -74,14 +72,29 @@
                 "default-namespace", "default"
             ),
             ORC,
-            formatVersion()
+            1
+        },
+        { "testhive", SparkCatalog.class.getName(),
```

Comment:
Just adding two more runs , should not take too long, and I think will make this test much more deterministic (right now randomly picks a version)